youtube_dl/extractor/youtube.py

```diff
@@ -1435,9 +1435,11 @@ def add_dash_mpd(video_info):
         else:
             age_gate = False
             video_info = None
+            sts = ''
             # Try looking directly into the video webpage
             ytplayer_config = self._get_ytplayer_config(video_id, video_webpage)
             if ytplayer_config:
+                sts = ytplayer_config['sts']
```

Comment:
I'm not a python dev, so I assume using the 'get' method of the ytplayer_config dictionary is correct.